src/binary_basic_decode.ml

```diff
@@ -0,0 +1,640 @@
+open Types
+open Syntax
+
+module Input = struct
+  type t =
+    { bytes : string
+    ; pt : int
+    ; size : int
+    }
+
+  let _pp input =
+    print_int input.pt;
+    print_string " ";
+    print_int input.size;
+    print_newline ()
```

Comment:
Could you either:
- base it on Format and remove the underscore,
- or remove it altogether?